client/python/vdms/__init__.py

```diff
@@ -0,0 +1,4 @@
+name = "vdms"
```

Comment:
For some reason I am seeing the python pull request merged with flip feature PR. Could you make sure they are separately merged on develop?

Reply:
this will go away, yes. it is here only because the python PR is also pending.